spec/time_spec.rb

```diff
@@ -5,8 +5,18 @@
 RSpec.describe Time do
   describe "#to_era" do
 
+    context 'time is 2019,7,2' do
+      subject { Time.mktime(2019,7,2) }
+      include_examples "2019,7,2"
+    end
+
+    context 'time is 2019,5,1' do
+      subject { Time.mktime(2019,5,1) }
+      include_examples "2019,5,1"
+    end
+
     context 'time is 2019,4,30' do
-      subject { Date.new(2019,4,30) }
+      subject { Time.mktime(2019,4,30) }
```

Comment:
I fixed this because it is a spec for time 🕐

Reply:
Thank you!

I overlooked it 😢